.github/workflows/test.yaml

```diff
@@ -22,18 +28,21 @@ jobs:
         # When updating the minimum Python version here, also update the
         # `python_requires` from `setup.cfg`.
         # Run on latest minor release of each major python version.
-        python-version: ["3.6.7", 3.8, 3.9, pypy-3.6, pypy-3.7]
+        python-version: [3.6.7, 3.7, 3.8, 3.9, 3.10.0-alpha.5, pypy-3.6, pypy-3.7]
         tox-env: [alldeps]
         noipv6: ['']
 
         include:
-          # Run nodeps tests and add some extra coverage for latest 3.6 micro
-          # version.
+
+          # `nodeps` is created to make sure we don't have import errors in the
+          # runtime and production code.
+          # Any supported Python version is OK for this job.
```

Comment:
```suggestion
          # The minimum supported Python version should be used to maximize
          # coverage of code that otherwise depends on backports.
```